sdks/java/container/build.gradle

```diff
@@ -68,6 +68,11 @@ golang {
   }
 }
 
+task copyDependencyLicenses(type: Copy) {
```

Comment:
Can this task be added to buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy in applyDockerNature?